### PR TITLE
Make Disqus and Flattr HTTPS friendly

### PR DESCRIPTION
**Pull request by emmanuelbernard**

### CODE REVIEW

File: lib/awestruct/extensions/disqus.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@ def disqus_comments()
             var disqus_identifier = #{identifier};
             (function() {
               var dsq = document.createElement("script"); dsq.type = "text/javascript"; dsq.async = true;
-              dsq.src = "http://#{site.disqus}.disqus.com/embed.js";
+              dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
```
Comment (LightGuard) [inline lib/awestruct/extensions/disqus.rb]: Where is `disqus_shortname` coming from? Shouldn't it be `site.disqus_shortname`?

File: lib/awestruct/extensions/disqus.rb [hunk 2]
```diff
@@ -46,7 +46,7 @@ def disqus_comments_count()
             var disqus_shortname = '#{site.disqus}';
             (function () {
               var s = document.createElement('script'); s.async = true;
-              s.src = "http://disqus.com/forums/#{site.disqus}/count.js";
+              s.src = '//' + disqus_shortname + '.disqus.com/count.js';
```
Comment (LightGuard) [inline lib/awestruct/extensions/disqus.rb]: Same here

File: lib/awestruct/extensions/disqus.rb (hunk 1 above)
Comment (emmanuelbernard) [inline lib/awestruct/extensions/disqus.rb]: You scared me for a minute. if you expend the diff, you will see that disqus_shortname is a javascript variable.

    var disqus_shortname = '#{site.disqus}';

File: lib/awestruct/extensions/disqus.rb (hunk 2 above)
Comment (emmanuelbernard) [inline lib/awestruct/extensions/disqus.rb]: Same here

Comment (LightGuard) [inline lib/awestruct/extensions/disqus.rb]: Ah :) my bad, I wasn't paying attention. Didn't notice this was JS.

On Wednesday, May 20, 2015, Emmanuel Bernard <notifications@github.com>
wrote:

> In lib/awestruct/extensions/disqus.rb
> <https://github.com/awestruct/awestruct/pull/474#discussion_r30752549>:
>
> > @@ -46,7 +46,7 @@ def disqus_comments_count()
> >              var disqus_shortname = '#{site.disqus}';
> >              (function () {
> >                var s = document.createElement('script'); s.async = true;
> > -              s.src = "http://disqus.com/forums/#{site.disqus}/count.js";
> > +              s.src = '//' + disqus_shortname + '.disqus.com/count.js';
>
> Same here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/awestruct/awestruct/pull/474/files#r30752549>.
>


-- 
Sent from Gmail Mobile

**Pull request merged**